common/presenters/move-to-card-component.js

```diff
@@ -44,6 +45,17 @@ function moveToCardComponent({
       })
     }
 
+    if (move.timeline_events) {
+      for (let i = 0; i < move.timeline_events.length; i++) {
+        const eventType = move.timeline_events[i].event_type
+
+        if (eventType === 'MoveLockout') {
+          showIsALockout = true
+          break
+        }
+      }
```

Comment:
```suggestion
      showIsALockout = move.timeline_events.some(({ event_type }) => event_type === 'MoveLockout')
```

[`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is a method on arrays and returns true if one of the elements of the array passes the function provided to it.

Reply:
Niceee!